src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/logger.html.twig

```diff
@@ -110,6 +110,8 @@
 
 
 {% macro display_message(log_index, log, is_deprecation) %}
+    {{ is_deprecation ? 'DEPRECATED':log.priorityName }} - {{ log.message }}
```

Comment:
I might be picky, but can you add spaces around `:` to make it a bit more readable? :)